### PR TITLE
Add description for Chronicle Queue

### PR DESCRIPTION
**Pull request by peter-lawrey**

### CODE REVIEW

File: projects.yml [hunk 1]
```diff
@@ -541,3 +541,27 @@ popular:
         url: https://engineering.groupon.com/2013/hornetq/building-a-distributed-messaging-system/
       - text: "Reliable Distributed Messaging with HornetQ"
         url: http://www.jboss.org/dms/judcon/2013unitedstates/presentations/judcon2013_day3track1session5.pdf
+  chronicle-queue:
+    name: Chronicle Queue
+    url: http://chronicle.software/products/chronicle-queue/
+    summary: "Persisted queue for recording all messages passed between threads or JVMs to build deterministic state machines. It supports sub-micro-second latencies between JVMs and millions of events per second with one machine. Clustered systems can handle bursts of tens of millions of events per second. Self describing messages allow for long term storage of data (years) even if the schema changes. License Apache v2.0"
```
Comment (JensRantil) [inline projects.yml]: `s/with one machine/within one machine`?

File: projects.yml [hunk 2]
```diff
@@ -541,3 +541,27 @@ popular:
...
+    tags:
+      - java
+      - C-sharp
```
Comment (JensRantil) [inline projects.yml]: Is it written both in Java _and_ C-sharp? Seems odd.

**Comment by peter-lawrey** [comment]
It is written in Java and has a C# client.

**Comment by peter-lawrey** [comment]
replaced "with" as "using"

**Comment by peter-lawrey** [comment]
I note other solutions list the technologies of their clients, or did I read this incorrectly?

File: projects.yml (hunk 1 above)
Comment (peter-lawrey) [inline projects.yml]: using is perhaps better

File: projects.yml (hunk 2 above)
Comment (peter-lawrey) [inline projects.yml]: other solution have listed the technology of their clients. There is a C# client.

**Comment by JensRantil** [comment]
> I note other solutions list the technologies of their clients, or did I read this incorrectly?

If that's how it's been done before I suggest we keep doing like that.

**Comment by peter-lawrey** [comment]
Admittedly, the generic "tags" could be more specific. Perhaps a clients tag...

**Comment by lukaszx0** [comment]
The site has been effectively abandoned for a long time now and I've always been a very bad steward of it. Also, I don't think it serves the purpose it used to serve so I'm archiving this repo and taking the site down. Sorry and thanks for your contributions!

**Pull request closed without merging**